### PR TITLE
DOC-280 add PROPCOUNT + reformat

### PR DESCRIPTION
**Pull request by markzegarelli**
Release around 10/10

### CODE REVIEW

**Review by SpencerFleury — APPROVED** [review]

File: content/collections/experiment-results/en/experiment-results-use-formula-metrics.md [hunk 1]
```diff
@@ -123,6 +123,18 @@ This function only works when grouping by a numerical property on the last event
...
 
+### PROPCOUNT
+
+**Syntax:** `PROPCOUNT(event)`
+
+* **Event:** Refers to the event that interests you. This must be a letter that corresponds to an event in the Events card. If grouping by multiple properties, the formula runs the calculation with the first group by clause.
+
+Returns the number of distinct property values for the property the event is grouped by. In this setup, the formula retrieves the number of different departments covering all the items for which details were viewed:
+
+![propcount_sidecontrols.png](/docs/output/img/event-segmentation/propcount-sidecontrols-png.png)
```
Comment (akhil-prakash) [inline content/collections/experiment-results/en/experiment-results-use-formula-metrics.md:134]: this is an image from an event seg chart which is a little weird to have that in an article that talks about experiment results chart

File: content/collections/experiment-results/en/experiment-results-use-formula-metrics.md [hunk 2]
```diff
@@ -123,6 +123,18 @@ This function only works when grouping by a numerical property on the last event
...
+
+`PROPCOUNT` is an **estimate** of distinct property values. This estimate comes from a [HyperLogLog algorithm,](https://en.wikipedia.org/wiki/HyperLogLog) and its accuracy depends on amount of data it has to work with. Expect a relative error in the range of 0.1% for less than 12,000 unique values, and up to 0.5% for more than 12,000 unique property values, depending on the cardinality of the property.
```
Comment (akhil-prakash) [inline content/collections/experiment-results/en/experiment-results-use-formula-metrics.md:136]: can you also add that we only support numeric event properties in experiment but analytics supports any data type as the event property you are counting

**Review by akhil-prakash — APPROVED** [review]
Just FYI: we have a bug bash for propcount on friday 10/11. So we will probably release on 10/15 since 10/14 is a holiday

**Review by akhil-prakash — APPROVED** [review]

**Comment by akhil-prakash** [comment]
@markzegarelli you can merge this whenever you have time

**Pull request merged**